data/json/items/comestibles/drink.json

```diff
@@ -190,6 +190,30 @@
     "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ],
     "fun": 6
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "cataclysm_coffee",
+    "name": "cataclysm coffee",
+    "name_plural": "cataclysm coffee",
+    "weight": 250,
+    "color": "brown",
+    "addiction_type": "caffeine",
+    "use_action": "CAFF",
+    "stim": 10,
```

Comment:
Isn't Kentucky coffee caffeine-free?

Reply:
I couldn't actually find any sources on whether or not it had any caffeine in it, only that it was used as a substitute for coffee by English settlers and Native Americans. If you can find something that mentions caffeine content either way, I'll remove the caffeine bit and drop the stims.

Reply:
I can't find any serious sources one way or another, but every source that does state anything about caffeine content says it has none.

Reply:
I figured that that would be the case, and I dug pretty deep. I guess not everyone is interested in the specifics of how a pod plucked off a tree works caffeine-wise.